src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -271,7 +271,10 @@ class DocBaseViewer extends BaseViewer {
         // the file. Users without download permissions shouldn't be able to
         // interact with the text layer
         const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
-        this.findBar = new DocFindBar(this.findBarEl, this.findController, canDownload);
+
+        if (this.getViewerOption('useFindBar') !== false) {
```

Comment:
I think this might be more clear as `disableFindBar`, since the default behavior is to use the find bar. Something like `if (!this.getViewerOption('disableFindBar') {` cc @tonyjin